scripts/rust_codegen.sh

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env bash
+
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+set -eEuo pipefail
+
+codegen () {
+    PROTO_DIR=$1
+    PROJECT_DIR=$2
+    OUT_DIR=$3
+
+    PROTOC_VERSION='24.4'
```

Comment:
I think it is only necessary to check the major version, as the minor ones do not introduce breaking changes. Furthermore, the latest release is not always immediately available on every system.

Reply:
see this comment https://github.com/astarte-platform/astarte-message-hub-proto/pull/17#discussion_r1356530734